### PR TITLE
Add client_user header into the Director's CORS config and update UI reference

### PR DESCRIPTION
**Pull request by danielgospodinow**
**Description**

This PR fixes the issue with SFSF/C4C application connection in the Compass UI.

Changes proposed in this pull request:
- Change the virtual service of the Gateway component to setup the `client_user` header in the Director's CORS configuration.
- Make the clientID header configurable in the Director.
- Update Compass UI reference.

**Pull Request status**

- [x] Implementation
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

**Review by alextargov — DISMISSED** [review]

**Review by alextargov — DISMISSED** [review]

File: chart/compass/charts/gateway/templates/virtualservice.yaml [hunk 1]
```diff
@@ -72,6 +72,7 @@ spec:
           - "authorization"
           - "content-type"
           - "tenant"
+          - "client_user"
```
Comment (NickyMateev) [inline chart/compass/charts/gateway/templates/virtualservice.yaml:75]: Don't forget to not leave the value hard coded here and in the other virtualservice file as well

**Review by NickyMateev — CHANGES_REQUESTED** [review]

**Review by NickyMateev — APPROVED** [review]

**Pull request merged**